src/test/resources/network/brightspots/rcv/test_data/test_set_1_exhaust_at_overvote/test_set_1_exhaust_at_overvote_config.json

```diff
@@ -15,23 +15,23 @@
     "idColumnIndex" : "",
     "precinctColumnIndex" : "",
     "provider" : "CDF",
-    "contestId": "contest-001"
+    "contestId": "test set 1"
   } ],
   "candidates" : [ {
-    "name" : "A",
-    "code" : "cs-a",
+    "name" : "Candidate A Name",
```

Comment:
Here I removed the internal cdf CandidateId usage, and changed the name to something that is more clearly a Name.